packages/full-stack-tests/src/hierarchy-builder/grouping/BaseClassGrouping.test.ts

```diff
@@ -0,0 +1,764 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { PhysicalPartition, Subject } from "@itwin/core-backend";
+import { IModel } from "@itwin/core-common";
+import { IModelConnection } from "@itwin/core-frontend";
+import { SchemaContext } from "@itwin/ecschema-metadata";
+import { ECSchemaRpcLocater } from "@itwin/ecschema-rpcinterface-common";
+import { createECSqlQueryExecutor, createMetadataProvider } from "@itwin/presentation-core-interop";
+import { HierarchyNode, HierarchyProvider, IHierarchyLevelDefinitionsFactory, NodeSelectClauseFactory } from "@itwin/presentation-hierarchy-builder";
+import { buildIModel, insertPhysicalPartition, insertSubject } from "../../IModelUtils";
+import { initialize, terminate } from "../../IntegrationTests";
+import { NodeValidators, validateHierarchy } from "../HierarchyValidation";
+
+describe("Stateless hierarchy builder", () => {
+  describe("BaseClassGrouping", () => {
```

Comment:
Full stack tests test situations rather than APIs, so `describe` blocks should be in proper language, e.g. "Base class grouping" here.

Reply:
Changed.